hack/vendor.sh

```diff
@@ -46,6 +46,7 @@ case $# in
 	;;
 esac
 
+clone git github.com/coreos/go-systemd/daemon
```

Comment:
`./hack/vendor.sh github.com/coreos/go-systemd` - the script will import the `daemon` sub-pkg correctly (hopefully)